waku/waku_relay/protocol.nim

```diff
@@ -122,7 +121,10 @@ const GossipsubParameters = GossipSubParams(
 type
   WakuRelayResult*[T] = Result[T, string]
   WakuRelayHandler* = proc(pubsubTopic: PubsubTopic, message: WakuMessage): Future[void] {.gcsafe, raises: [Defect].}
+  WakuValidatorHandler* = proc(pubsubTopic: PubsubTopic, message: WakuMessage): Future[ValidationResult] {.gcsafe, raises: [Defect].}
   WakuRelay* = ref object of GossipSub
+    wakuValidators*: Table[string, seq[WakuValidatorHandler]]
+    validatorInserted*: Table[string, bool]
```

Comment:
These can be private.
Can we also have a short line describing them a little? Specially interesting the expected values for Keys, or maybe explicitly set `Pubsubtopic` as a type for the keys.
```suggestion
    wakuValidators: Table[string, seq[WakuValidatorHandler]]
    validatorInserted: Table[string, bool]
```